scenes/game/game.gd

```diff
@@ -1520,16 +1520,30 @@ func _on_exceed_revert_event(event):
 func _on_become_wide(event):
 	var player = event['event_player']
 	if player == Enums.PlayerId.PlayerId_Player:
-		$PlayerCharacter.is_wide = true
+		var player_character = $PlayerCharacter
+		player_character.is_wide = true
 		if 'wide_animation' in player_deck:
-			$PlayerCharacter.load_character(player_deck['wide_animation'])
-			move_character_to_arena_square($PlayerCharacter, game_wrapper.get_player_location(Enums.PlayerId.PlayerId_Player), true, Character.CharacterAnim.CharacterAnim_None)
+			player_character.load_character(player_deck['wide_animation'])
+
+			var parent = player_character.get_parent()
+			var opponent_buddy_idx = parent.get_children().find(opponent_buddies[0])
```

Comment:
I guess this isn't an issue, but if we ever added a wide character that also had buddies, it should be above both players buddies.
But that doesn't exist, so it's fine.

Reply:
Though I do feel like there is a lot of code duplication going on in this function.
You could probably simplify it by setting up all the variables in the if and then doing the logic afterwards.
If you do that, you could also just add a node for where wide characters go and add the character to that instead of putting it where we put buddies.